project/src/main/ui/overworld-ui.gd

```diff
@@ -119,7 +120,7 @@ Returns:
 	The center of the smallest rectangle including all characters in the current conversation. If no characters meet
 	this criteria, this returns an zero-size rectangle at (0, 0).
```

Comment:
This method comment is no longer accurate.

Reply:
Fixed method comment, `(Returns) the smallest rectangle including all characters...`